scripts/deterministic-slide/convert.py

```
@@ -0,0 +1,756 @@
+        elif t == "Code":
+            # Code は [attrs, str]
+            parts.append(f"<code>{html.escape(c[1])}</code>")
+        elif t == "Link":
+            attrs, inner, target = c
+            url, _title = target
+            url = unwrap_google_redirect(url)
+            # リンクの内側テキストは既に <a> の中なので二重にラップしない
+            inner_html = render_inlines(inner, images_b64, autolink=False)
+            parts.append(f'<a href="{html.escape(url)}">{inner_html}</a>')
+        elif t == "Image":
+            attrs, alt_inlines, target = c
```

Ruff (0.15.20)

[warning] Unpacked variable `attrs` is never used

(RUF059)